src/tools/generate/generate_chapters.js

```diff
@@ -2,7 +2,10 @@ const fs = require('fs-extra');
 const showdown = require('showdown');
 const ejs = require('ejs');
 const prettier = require('prettier');
+const rainbow = require('rainbow-code');
```

Comment:
Could we move all this code into it's own file? Like we do for generate_header_links, generate_figure_ids...etc.